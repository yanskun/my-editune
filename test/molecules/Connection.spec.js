import { createLocalVue, shallowMount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import ElementUI from "element-ui"
import Component from "~/components/molecules/Connection"

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
localVue.use(ElementUI)

describe("Connection", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        icons: [
          {
            src: "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png",
            alt: "GitHub",
            href: "https://github.com/"
          }
        ]
      },
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
