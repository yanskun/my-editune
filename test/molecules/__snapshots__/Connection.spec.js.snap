// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connection snapshot 1`] = `
<div
  class="connection"
>
  <div
    class="icons"
  >
    <div
      class="icon"
    >
      <a
        href="https://github.com/"
        target="_blank"
      >
        <el-image-stub
          alt="GitHub"
          class="image"
          previewsrclist=""
          src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png"
          zindex="2000"
        />
      </a>
    </div>
  </div>
</div>
`;
